about.html

```diff
@@ -12,62 +12,85 @@
 </head>
 
 <body>
+  
+  <header>
+  <nav>
+      <img src="img/lambda-black.png" alt="">
+      <a href="index.html">Home</a>
+      <a href="about.html">About</a>
+      <a href="/">Products</a>
+      <a href="/">Blog</a>
+      <a href="/">Contact</a>
+    </nav>
+    <div><img src="img/jumbo-about.png" alt="" class="header-img"></div>
+  </header>
+  
     <div class="container about-page">
      
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
+      <div class="about">
+      <h2>About</h2>
 
+    <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+      </div>
+    <div class="working-pics">
+    <div class="work-divs">
     <img src="img/about-plan.png" alt="strategy">
     
-    Strategy
+    <h2>Strategy</h2>
     
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
+    <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
 
+    <a href="/">Learn More</a>
+    </div>
 
 
+    <div class="work-divs">
     <img src="img/about-working.png" alt="strategy">
```

Comment:
You have the right idea but maybe forgot about it, the alt attribute in each img tag is set to "strategy". For accessibility purposes, it's recommended that the attribute reflect the sections more accurately.